api/src/services/observation-services/observation-service.ts

```diff
@@ -57,21 +49,47 @@ export interface InsertSubCount {
     measurement_id: string;
     measurement_value: number;
   }[];
-  qualitative_environments: {
-    environment_qualitative_id: string; // uuid
-    environment_qualitative_option_id: string;
-  }[];
-  quantitative_environments: {
-    environment_quantitative_id: string; // uuid
-    value: number;
-  }[];
 }
 
 export type InsertUpdateObservations = {
   standardColumns: InsertObservation | UpdateObservation;
   subcounts: InsertSubCount[];
 };
 
+export type InsertObservations = {
```

Comment:
question: Where did we land with this, is this still being used or do we only need `InsertUpdateObservations`

Reply:
I believe we will eventually remove the `InsertUpdate` function, in favour of separate insert and update functions.

But, we haven't implemented the new edit observation flow yet. We will probably add it to this PR, or a feature branch, before merging any of this to Dev.